aten/src/ATen/native/cuda/UnaryComplexKernels.cu

```diff
@@ -24,12 +24,38 @@ __host__ __device__ static inline c10::complex<T> angle_wrapper(c10::complex<T>
   return std::arg(v);
 }
 
+const char angle_name[] = "angle_kernel";
 void angle_kernel_cuda(TensorIteratorBase& iter) {
-  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(iter.common_dtype(), "angle_cuda", [&]() {
+  auto dtype = iter.common_dtype();
+  if(isComplexType(dtype)) {
+#if AT_USE_JITERATOR()
+  static const auto angle_string = jiterator_stringify(
+      template <typename T>
+      T angle_kernel(T v) {
+          return std::arg(v);
+      }
+  ); // angle string
+  AT_DISPATCH_COMPLEX_TYPES(dtype, "angle_cuda", [&]() {
```

Comment:
Can we also add `chalf` support here?

Reply:
Yep! I was wondering about the same. Thanks, Kshitij! 
Will add `chalf` support! :) That's pretty interesting!